src/js/main.js

```diff
@@ -0,0 +1,19 @@
+// TODO: Make buttons click over to corresponding tabs.
+// TODO: Convert buttons into tabs.
+// TODO: Add event listener 'click' to each button that changes class to active.
+//
+// For now make the tab 'repo' always active.
+
+$(document).ready(function(){
```

Comment:
This looks a lot like the jQuery found here: http://inspirationalpixels.com/tutorials/creating-tabs-with-html-css-and-jquery#step-jquery

Always attribute your source!! And be sure you modify the code appropriately to work with your current project.